@shared/api/external.ts

```diff
@@ -20,9 +19,9 @@ export const requestPublicKey = async (): Promise<string> => {
   return publicKey;
 };
 
-export const submitTransaction = async ({
-  transactionXdr,
-}: LyraApiRequest): Promise<string> => {
+export const submitTransaction = async (
```

Comment:
Might be nice to rename this function too?

Reply:
I purposely didn't call this `signTransaction` because I didn't want 2 functions with that name in the codebase. 

Also, the functions in this file are just passing off lyra-api requests to Background and mutating the response. The names are supposed to signify they're "attempting" the action called by lyra-api